optimum/exporters/openvino/model_patcher.py

```diff
@@ -444,9 +444,10 @@ def _qwen_attention_forward(
             value = value.permute(0, 2, 1, 3)
 
         if not self.use_cache_quantization and SUPPORT_SDPA:
-            causal_mask = registered_causal_mask[:, :, key.size(-2) - query.size(-2) : key.size(-2), : key.size(-2)]
+            # For performance, using constant tril to generate causal_mask
+            causal_mask = self.bias[:, :, key.size(-2) - query.size(-2) : key.size(-2), : key.size(-2)]
```

Comment:
did you check is it work with the latest qwen version?

Reply:
Yes, I test the patch with the latest Qwen-7b-chat model (commit `93a65d34827a3cc269b727e67004743b723e2f83`). It works well. The usage of SDPA remains the same between commit `2abd8e5777bb4ce9c8ab4be7dbbd0fe4526db78d` and `93a65d34827a3cc269b727e67004743b723e2f83`